compiler/visq/visq

```diff
@@ -198,6 +203,18 @@ def main():
             with open(args.mpeir_output, "w") as f:
                 json.dump(result, f)
 
+        if args.mse_output:
+            computer = MSEComputer(fp32_dir, fq_dir)
+            qerror_map, q_min, q_max = computer.run()
+            palette = YLORRD9Palette(qerror_min=q_min, qerror_max=q_max)
+            result = _build_json(
+                metric='MSE',
+                model=fp32_model,
```

Comment:
it is not `Path(fp32_model).stem`?

Reply:
viewer assumes model path is relative to this json or absolute path is also ok.

Reply:
I got it but above `mpeir_output` case uses `Path(fp32_model).stem`. Is there any difference?

Reply:
Ah, I misunderstood the spec of meta section. Actually it should be `Path(..).name`. Ex: "test.circle"

Path(fp32_model).name -> "test.name"
Path(fp32_model).stem -> "test"

stem will be updated in the next PR. Thanks for finding!